pages/common/theHarvester.md

```diff
@@ -0,0 +1,24 @@
+# theHarvester
+
+> A tool designed to be used in the early stages of a penetration test.
+> More information: <https://github.com/laramies/theHarvester>.
+
+- Gather information on a domain using Google:
+
+`theHarvester --domain {{domain_name}} --source google`
+
+- Gather information on a domain using multiple sources:
+
+`theHarvester --domain {{domain_name}} --source google,bing,crtsh`
+
+- Change the limit of results to work with:
+
+`theHarvester --domain {{domain_name}} --source google --limit 200`
+
+- Save the output to an xml and html file:
+
+`theHarvester --domain {{domain_name}} --source google --file {{output_file}}`
```

Comment:
Missing token syntax? Also, perhaps this would be more consistent with other existing pages?
```suggestion
`theHarvester --domain {{domain_name}} --source {{google}} --file {{path/to/output_file}}`
```

Reply:
I'm not sure how to write out the file format. You can provide a path, but for ex. when you provide outputdir/outputfile it will save a file called outputfile.xml and one called outputfile.html in the directory outputdir. So because you don't provide an extension it is kind of confusing.